pages/pipelines/create_your_own.md

```diff
@@ -91,6 +91,17 @@ If you've configured webhooks, your pipeline will trigger when you push updates
 
 If you have trouble getting your pipeline to work, don't hesitate to reach out to [support](https://buildkite.com/support) for help.
 
+### On slug creation conventions
+
+Pipeline slugs are generated based on the pipeline name you provide during the pipeline creation.
+The maximum allowed character length for a pipeline slug is `100`.
+The supported character format for slug generation is:
+`/\A[a-zA-Z0-9]+[a-zA-Z0-9\-]*\z/`.
+
+All the whitespace characters that appear in the pipeline name consecutively will be converted to a single `-` character (so `"Hello[space]there"` and `"Hello[space, space, space, etc.]there"` will be equally converted to a `hello-there` slug), and uppercase will be converted to lowercase.
+
+An attempt at creating a new pipeline with a name that matches an existing pipeline name will throw an error.
+
```

Comment:
For all the changes to make to `pages/apis/graphql/cookbooks/pipelines.md` above, make/reflect them in this file too.

Also, I'm not quite sure if this is the right place to include this text. Maybe below the 'using private repositories', since this section seems more important and would be better placed above the section on deriving slugs?

Reply:
Done.